src/vuestic-theme/vuestic-components/va-input/VaInputWrapper.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div
+    class="va-input-wrapper d-flex align--end text--left"
+    :class="{ 'va-input-wrapper--disabled' : disabled }"
+  >
+    <div class="va-input-wrapper__control">
+      <div
+        tabindex="0"
+        :style="slotStyles"
+        class="va-input-wrapper__slot d-flex align--end pt-2">
+        <div
+          v-if="hasPrependData"
+          class="va-input-wrapper__prepend-inner d-inline-flex align--center ml-2">
+          <slot name="prepend"/>
+        </div>
+        <div>
+          <slot/>
+        </div>
+        <div
+          v-if="hasAppendData"
+          class="va-input-wrapper__append-inner d-inline-flex align--center mr-2">
+          <slot name="append"/>
+        </div>
+      </div>
+      <div class="va-input-wrapper__details py-0 px-2">
+        <div class="va-input-wrapper__messages">
+          <div
+            v-if="error"
+            :style="messageStyles"
+            class="va-input-wrapper__messages__wrapper">
+            <template
+              v-for="message in errorMessages">
+              {{ message }}
+            </template>
+          </div>
+          <div
+            v-else
+            :style="messageStyles"
+            class="va-input-wrapper__messages__wrapper">
+            <template v-for="message in messages">
+              {{ message }}
+            </template>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'va-input-wrapper',
+  props: {
+    disabled: {
+      type: Boolean
+    },
+    error: {
+      type: Boolean
+    },
+    success: {
+      type: Boolean
+    },
+    messages: {
+      type: Array
```

Comment:
Why do you not define default values in arrays? It will be error, if we forget pass them as a prop.